### ISSUE TITLE
View details of the complaint on the Complaint details page

### ISSUE THREAD

**Issue by lalitha-jonnalagadda** [18F/crt-portal#186]
## Narrative
As a CRT User,
I want to view additional details of a complaint record
so that I can confirm if the complaint has merit

## Background
The complaint details allows DOJ CRT to determine whether they have jurisdiction to act over a complaint. 

## Assumptions
- From the details page, user can not navigate to other complaints (detailed pages of other complaints) using the 'Next' and 'Back' buttons
- Modular boxes are not collapsible or expandable
- View complainant information on detail page has been developed

## Out of scope
- Changing statuses
- (Re)Routing complaints
- Editing a complaint

## Acceptance criteria
1) Given I am viewing a complaint on the view all complaints table
When I click on a complaint (row)
 Then I am directed to that complaint's record page
2) Given I've clicked on a complaint in the view all table
When I am viewing that complaint's record page
Then I view a new section under 'Correspondent Information' called 'Complaint Details'
AND
I see the following information: 
- [ ] Primary Issue
- [ ] Hate Crime
- [ ] Human trafficking
- [ ] Relevant Details
- [ ] Location Name
- [ ] City/State
- [ ] Protected Class
AND
information I view on that record details page(city/town, state and protected class(es) matches the information of that complaint line item the view all page 

3) Given that I'm on the complaint details page
When I'm on the complaint details section
Then I see Primary issue selection is displayed in bold

4) Given that a complainant is on the 'hate crime and human trafficking' question
When they check either hate crime/human trafficking options
Then I see hate crime and human trafficking fields display 'yes(checked)' 
AND 
When these options are unchecked, 
Then I see the fields display 'Not specified'

5) Given that I'm on the complaint details section
When I'm on the Protected classes field
AND 
When I'm on the relevant details field
Then I see the question a public user answered along with the corresponding selection (see mockup for example)

6) A user should be able to navigate through this page using a keyboard following a tab order that matches what's visually displayed (through the three links/buttons)

7) As a screen reader/low vision user....
- [ ] I would expect semantic headings for Complaint details (H2) , Correspondent information (H3), Complaint details (H3), Complaint summary (H3)
 - [ ] I'm not sure how it's built, but I would expect to be able to associate data with the label (Using a table or maybe they're grouped) -- appears to be working this way now
- [ ]  Button uses standard component, the whole area is a touch target appears to be working this way now
- [ ] be able to resize page to 200% without needing bi-directional scrolling (overflow works)
- [ ] Proper contrast ratios for all text & informative icons and buttons that aren't disabled (should be built into the design)
 

## Mockups
![DETAILS - V1, info we have.jpg](https://images.zenhubusercontent.com/5d1a3b0bcc794602e102a513/2c2d46cb-45a9-4b59-9a3b-5a1dbc59a022)

## Sign off
- [x] PO
UAT sign off
- [ ] PO

**Comment by Jacklynn** [comment]
@lalitha-jonnalagadda Added the mock and made some changes based on co-design and the fact that this _won't_ be shared with the review page.

* Name is shared on a one line, last name first name and bolded like in @AvivaOskow's original
* hate crime & human trafficking questions have their own line since they can be summarized
* added conditional logic question (labeled "relevant details" in the mock but can be changed to whatever)

Design note: Aviva had some different styling for the correspondent info where the text size is slightly larger, but I have it all the same size for now.

**Comment by Jacklynn** [comment]
Updated a11y criteria

Suggesting the text for be 'Not specified' and updated the ACs to say that

**Comment by LindsayYoung** [comment]
Suggestion for an enhancement- we might want to rename "RELEVANT DETAILS" to "ELECTION DETAILS" and only show it if there is a value for it, since most sections will never see any answers to that field, and that could get confusing with a vague name.

**Comment by blaughman** [comment]
Complaint detail page works as expected in IE 11.  Moving to UAT.

**Comment by lalitha-jonnalagadda** [comment]
@el-mapache When I check this page on my laptop(smaller screen), primary issues that are lengthier get wrapped slightly oddly. Def NOT a blocker, but I wanted to share incase its a trivial change. 
 ![Screen Shot 2020-01-02 at 2.49.44 PM.png](https://images.zenhubusercontent.com/5b34455c5b51b10bd4dd59d6/61a05aae-a2f0-43a9-94e0-f84560bbca68)

**Comment by el-mapache** [comment]
I _think_ this should be trivial; we should be able to have the css break this line on a word!

**Comment by lalitha-jonnalagadda** [comment]
Will close this card after #257 is completed.

**Comment by blaughman** [comment]
Function works as intended.  Moving to UAT.

**Comment by Jacklynn** [comment]
@blaughman @lalitha-jonnalagadda I still don't see 'protected class' on the details page after hard refresh

**Comment by lalitha-jonnalagadda** [comment]
Showcased during deep-dive on 2/6.
